frame/election-provider-multi-phase/src/mock.rs

```diff
@@ -352,6 +351,10 @@ impl multi_phase::weights::WeightInfo for DualMockWeightInfo {
 	}
 }
 
+parameter_types! {
+	pub const RandomBalancing: Option<(usize, ExtendedBalance)> = Some((0, 0));
```

Comment:
```suggestion
	pub const Balancing: Option<(usize, ExtendedBalance)> = Some((0, 0));
```

It is anything but random, to be pedantic :D